### ISSUE TITLE
Objects unable to be loaded should be ignored and reported to user

### ISSUE THREAD

**Issue by janisozaur** [OpenRCT2/OpenRCT2#6074]
**Commit/Build:** 0d52030b4

Right now we detect invalid objects and call out `assert` if they do something weird, e.g.:

```
$ ./openrct2
Object repository is out of date.
Scanning 4566 objects...
openrct2: /home/janisozaur/workspace/OpenRCT2/src/openrct2/util/sawyercoding.c:251: decode_chunk_rle_with_size: Assertion `dst + count <= dst_buffer + dstSize' failed.
```

Rather than doing that, we should be able to just ignore an object and report broken file to the user.

You can use https://github.com/OpenRCT2/OpenRCT2/files/1184445/DISKO.txt file reported in #6066.

**Comment by IntelOrca** [comment]
@janisozaur We now have the object load error window, can this be closed?

**Comment by janisozaur** [comment]
Yes, it works now.

If you find the commit/PR it would be best.

**Comment by IntelOrca** [comment]
Fixed in #5624
